src/Command/SearchClearCommand.php

```diff
@@ -23,20 +33,28 @@ protected function configure()
             );
     }
 
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $indexToClear = $this->getEntitiesFromArgs($input, $output);
 
         foreach ($indexToClear as $indexName => $className) {
-            $success = $this->indexManager->clear($className);
+            $success = $this->searchService->clear($className);
 
-            if ($success) {
-                $output->writeln('Cleared <info>'.$indexName.'</info> index of <comment>'.$className.'</comment> ');
+            if ($success instanceof IndexingResponse) {
+                $output->writeln('Cleared <info>' . $indexName . '</info> index of <comment>' . $className . '</comment> ');
             } else {
-                $output->writeln('<error>Index <info>'.$indexName.'</info>  couldn\'t be cleared</error>');
+                $output->writeln('<error>Index <info>' . $indexName . '</info>  couldn\'t be cleared</error>');
             }
         }
 
         $output->writeln('<info>Done!</info>');
+
+        return null;
```

Comment:
Is it expected to return `null`? My Java/C# experience is telling me that return such a value could lead to lots of run time exceptions! Wouldn't be possible to make those functions `void`? 

If I am not wrong `void` is available in PHP 7.

Reply:
Indeed, here you don't need to return nothing.